backend/libs/config/src/rmqConfiguration.ts

```diff
@@ -0,0 +1,19 @@
+import { RmqOptions, Transport } from '@nestjs/microservices';
+import { RmqQueue } from '@app/types/rmqQueues';
+
+const getRmqOptionsForQueue = (rmqQueue: RmqQueue): RmqOptions => {
+  const rmqUrl = process.env.RMQ_URL;
+  return {
+    transport: Transport.RMQ,
+    options: {
+      urls: [rmqUrl],
+      queue: rmqQueue,
+    },
+  };
+};
+
+export const getRmqOptions = () => ({
+  questionServiceOptions: getRmqOptionsForQueue(RmqQueue.QUESTION),
+  authServiceOptions: getRmqOptionsForQueue(RmqQueue.AUTH),
+  userServiceOptions: getRmqOptionsForQueue(RmqQueue.USER),
+});
```

Comment:
I think we can have a 
```
const RmqOptions = {
  questionService: getRmqOptionsForQueue(RmqQueue.QUESTION),
  authService: ...,
  userService: ...
}
```
since the queue options will never change. But i suspect this will fail since the config object has yet to be loaded.

Alternatively, we can just call `getRmqOptionsForQueue` in the microservice directly